cmake/msvc_utils.cmake

```diff
@@ -0,0 +1,23 @@
+macro(configure_msvc_runtime)
+    if(MSVC)
+        set(MSVC_RUNTIME_STATIC ON CACHE BOOL "Static linking of MSVC runtime libraries.")
+        if(${MSVC_RUNTIME_STATIC})
+            message(STATUS "MSVC -> Forcing use of statically-linked runtime.")
+            foreach(flag_var
+                    CMAKE_C_FLAGS
+                    CMAKE_C_FLAGS_DEBUG
+                    CMAKE_C_FLAGS_MINSIZEREL
+                    CMAKE_C_FLAGS_RELEASE
+                    CMAKE_C_FLAGS_RELWITHDEBINFO
+                    CMAKE_CXX_FLAGS
+                    CMAKE_CXX_FLAGS_DEBUG
+                    CMAKE_CXX_FLAGS_MINSIZEREL
+                    CMAKE_CXX_FLAGS_RELEASE
+                    CMAKE_CXX_FLAGS_RELWITHDEBINFO)
+                    string(REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
```

Comment:
why do we need static runtime for onecore? Where is it written?

Reply:
This is workaround to remove msvcp140.dll, vcruntime140.dll from one core compliance validation.

Reply:
Does it mean that we should use static runtime only for Windows SDK version older than "X" version, where "X" version has `BinaryExclusionlist.xml` list for apiValidator? As far as I can see versions older "10.0.20000.0" does not have `BinaryExclusionlist.xml`, but newer versions have

Reply:
BTW, in the current master branch we have `apiValidator` run as `POST_BUILD` step for some OpenVINO dlls. But this run takes into account that `msvcp140.dll`, `vcruntime140.dll` libraries are exclusions even for `10.0.1XYZW.0` versions